### PR TITLE
feat: add analytics data platform icon

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
Icon for ADP added

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "ovh-manager-webfont",
     "description": "Managers front",
-    "version": "1.1.0",
+    "version": "1.2.0",
```
Comment (marie-j) [inline bower.json]: This doesn't need to be done manually and shouldn't

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ovh-manager-webfont",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (marie-j) [inline package.json]: Same here

File: bower.json (hunk 1 above)
Comment (ganeshkumar1989) [reply bower.json]: This has been undone

File: package.json (hunk 2 above)
Comment (ganeshkumar1989) [reply package.json]: This has been undone :)

**Pull request merged**